### ISSUE TITLE
single images or multiple images?

### ISSUE THREAD

**Issue by jumpjack** [phaseIV/Blender-Navcam-Importer#8]
From the screenshot in readme it looks like the plugin can combine multiple images into one single model; how do you accomplish this? I am working to a [javascript tool](https://github.com/jumpjack/VST-converter) which creates 3d models from .vst files, but each file represents one image, and I don't understand how to combine them into one single big model.
I don't know python.

**Comment by phaseIV** [comment]
Multiple models can be generated in Blender by running the addon multiple times or you can use the batch import option.
Both methods will add or merge the models into the current Blender scene.

**Comment by jumpjack** [comment]
Yes, I understand it,  but I would like to implement such feature in my script too: which metadata do you use from the images? I can see camera alt/az and rover motion counter, but I can’t see exact rover position.

Inviato da Posta per Windows

Da: phaseIV
Inviato: martedì 12 luglio 2022 12.19
A: phaseIV/Blender-Navcam-Importer
Cc: jumpjack; Author
Oggetto: Re: [phaseIV/Blender-Navcam-Importer] single images or multipleimages? (Issue #8)

Multiple models can be generated in Blender by running the addon multiple times or you can use the batch import option.
Both methods will add or merge the models into the current Blender scene.

**Comment by phaseIV** [comment]
The position data is represented in the mesh vertices. For each site that the rover has visited, all corresponding models will fit together. When the rover enters a new site the position data in the models will be reset.

**Comment by phaseIV** [comment]
You can find more data for each image at the nasa website [pds-imaging.jpl.nasa.gov](https://pds-imaging.jpl.nasa.gov)
When you check the label files it contains heaps of data to work with! See for example
[[pds-imaging.jpl.nasa.gov](https://pds-imaging.jpl.nasa.gov/data/msl/MSLNAV_0XXX/DATA/SOL00001/NLA_397586928ECS_N0010008AUT_04096M1.LBL)](https://pds-imaging.jpl.nasa.gov/data/msl/MSLNAV_0XXX/DATA/SOL00001/NLA_397586928ECS_N0010008AUT_04096M1.LBL)

**Comment by jumpjack** [comment]
Yes, I know very well the LBL files (for MERs), but I was in trouble figuring out the coordinates of each site.
Now I was eventually able to download  numerical raw data for the traverse maps, so now I should be able to properly position each model on Mars, see here: https://space.stackexchange.com/questions/59687/how-to-retrieve-mer2-spirit-rover-orientation-along-the-traverse-map/59706#59706
